### DISCUSSION TITLE
New database created but not displayed

### DISCUSSION THREAD

**Discussion by THE1960** [dbeaver/dbeaver#15121] [Q&A]
I have a connection on PostgreSQL. PostgreSQL-DB exists. I created one new database with a table and filled it with data. Then the program closed. After I restarted the program, the new DB was gone. Connection to PostgreSQL works. If I want to create a new one with this name, I become the message "Cannot create DB already exists ...". But I can't load or see them anywhere. The problem is reproducible.
Can you help me.

- The system is a new iMac 27' 2020 Intel
- I use postgres.app as postgresql

<img width="1918" alt="dbeaver problem 2022-01-05 um 14 50 41" src="https://user-images.githubusercontent.com/30343281/148228390-b7c83a40-3982-47a5-9034-fa324ab04fe2.png">

**Comment by LonwoLonwo** [comment]
Hello @THE1960 

Please check your connection settings with right click on your connection in the Database Navigator Tree.

The setting "Show all databases" must be enabled, if you want to see all your databases in one connection.

![2022-01-06 09_31_57-DBeaver Ultimate 21 3 2 - _postgres 13 3_Script](https://user-images.githubusercontent.com/45152336/148339586-a9b58684-f60f-48db-8175-d10387d0cfca.png)

**Comment by THE1960** [comment]
Good morning and wish you a good new year,

Thank you for the information. It works very well. I didn't know that I had to change these settings.

Freundliche Grüsse
Tobias Heise

Tobias Heise 			
Alte Torkelbergstr. 37	Mobile: +49 176 22749419
78465 Konstanz		Mail: ***@***.*** ***@***.***>

**Comment by ashwallen** [comment]
Currently the "show all databases" is moved in the Main section. It is just under your port tab.
